synchronization/src/androidTestCyface/java/de/cyface/synchronization/SyncAdapterTest.java

```diff
@@ -51,24 +52,43 @@
  *
  * @author Armin Schnabel
  * @author Klemens Muthmann
- * @version 2.2.3
+ * @version 2.2.4
  * @since 2.4.0
  */
 @RunWith(AndroidJUnit4.class)
+// To execute notice errors with the short running testOnPerformSync before the large
+// testOnPerformSyncWithLargeMeasurement which can save a lot of time
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
when we execute the test from the IDE the large test (~7-10 minutes) would be executed before the small one. Usually I only want to execute the small one as I know that the large one is executed on the CI anyway and it only tests that no OOM is produced during upload which should seldom break.

So this is a fast fix to execute the small test first and then I can cancel the other one if I want to.